src/framework/application.js

```diff
@@ -2162,6 +2170,7 @@ var makeTick = function (_app) {
         application._fillFrameStats();
         // #endif
 
+        this._inFrameUpdate = true;
```

Comment:
have this variable in the constructor as well please, to not introduce variables to class during running.